src/rules/noImplicitDependenciesRule.ts

```diff
@@ -77,7 +95,7 @@ function walk(ctx: Lint.WalkContext<Options>) {
     for (const name of findImports(ctx.sourceFile, ImportKind.All)) {
         if (!ts.isExternalModuleNameRelative(name.text)) {
             const packageName = getPackageName(name.text);
-            if (builtins.indexOf(packageName) === -1 && !hasDependency(packageName)) {
+            if (options.whitelist.indexOf(packageName) === -1 && builtins.indexOf(packageName) === -1 && !hasDependency(packageName)) {
```

Comment:
it would be a good idea to create a `Set` outside of the for loop to make the check faster.

Reply:
Done